04-NodeJS-NPM-Modules/topic4.js

```diff
@@ -0,0 +1,10 @@
+var movie = require('./modules/Movie').movie;
+var director = require('./modules/Director').director;
+require('./modules/jquery-2.1.4');
```

Comment:
Again, you might need to store its returned value, for instance: $ = require('./modules/jquery-2.1.4');
Like that, you can start using jQuery with $, as it is usually done.